requirements.txt

```diff
@@ -22,7 +22,7 @@ isort>=4.3.21
 jinja2>=2.11.1
 jsonschema>=3.2.0
 mergedeep>=1.3.0
-mypy==0.782
+mypy==0.910
```

Comment:
need to make sure this is aligned with content repo

Reply:
We try to remove it in Content repo and see the impact, as Mypy run in the linter in demisto-sdk and not directly